frontend/src/components/input/TextEditor.vue

```diff
@@ -275,6 +276,14 @@ const buttons = computed(() => [
     isCollapsable: false,
     callback: () => editor.chain().focus().toggleBlockquote().run(),
   },
+  {
+    type: "borderNumber",
+    icon: MaterialSymbolsDeleteSweepOutline,
+    ariaLabel: "borderNumber",
+    group: "format",
```

Comment:
Elemente in einer Group haben glaube ich keinen Trennstrich zwischen den Icons. Ich vermute, dass wir Bordernumber aber von den Blockqoutes trennen wollen?

Reply:
Done 👍